### PR TITLE
Pomodoro feature (inside parrot box)

### PR DESCRIPTION
**Pull request by gauravchl**
Commands:

```
p - Switch parrot to pomodoro mode
s - start/pause/resume pomodoro
e - stop pomodoro
u - update pomodoro duration
b - update break time
```

screen shot 2017-04-27 at 1 30 50 pm

### CODE REVIEW

**Comment by notwaldorf** [comment]
This looks really good!

File: care.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 var config = require(__dirname + '/config.js');
 var twitterbot = require(__dirname + '/twitterbot.js');
+var notifier = require('node-notifier');
+var Pomodoro = require(__dirname + '/pomodoro.js');
```
Comment (gauravchl) [inline care.js]: Sure 👍

File: care.js [hunk 2]
```diff
@@ -27,6 +31,44 @@ screen.key(['r', 'C-r'], function(ch, key) {
   tick();
 });
 
+screen.key(['s', 'C-s'], function(ch, key) {
+  if (!inPomodoroMode) return;
```
Comment (gauravchl) [inline care.js]: Yep 👍

File: pomodoro.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+// options.onBreakStarts (function) - Runs when break starts
+// options.onBreakEnds (function) - Runs when break ends
+var Pomodoro = function(options) {
+  var _setIntervalId = null;
+  var _runningDuration = 20; // Default pomodoro duration: 20 Min
+  var _breakDuration = 5;    // Default break duration: 5 Min
+  var _runningDurationRemaining = 0; // In seconds
+  var _breakDurationRemaining = 0;   // In seconds
+  var _currentState = States.STOPPED;
+
```
Comment (gauravchl) [inline pomodoro.js]: Sure :)

**Comment by gauravchl** [comment]
@notwaldorf Changes applied, Please review.
Thanks.

File: care.js (hunk 1 above)
Comment (gauravchl) [inline care.js]: @notwaldorf https://github.com/notwaldorf/tiny-care-terminal/pull/73/commits/7c67cd8603b940528287b1c22ed7a9b23cca56e6 What do you think  ?

**Comment by notwaldorf** [comment]
Awesome! Thanks so much!

**Pull request merged**